includes/bp-xprofile/classes/class-bp-rest-xprofile-field-groups-endpoint.php

```diff
@@ -861,6 +866,15 @@ public function get_collection_params() {
 			'validate_callback' => 'rest_validate_request_arg',
 		);
 
+		$params['include_groups'] = array(
+			'description'       => __( 'Ensure result set inludes specific profile field groups.', 'buddypress' ),
+			'default'           => array(),
+			'type'              => 'array',
+			'items'             => array( 'type' => 'integer' ),
+			'sanitize_callback' => 'bp_rest_sanitize_string_list',
```

Comment:
Since this is an array of ids...

```suggestion
			'sanitize_callback' => 'wp_parse_id_list',
```